filament/backend/src/metal/MetalHandles.h

```diff
@@ -164,7 +164,7 @@ class MetalRenderTarget : public HwRenderTarget {
     };
 
     MetalRenderTarget(MetalContext* context, uint32_t width, uint32_t height, uint8_t samples,
-            Attachment colorAttachments[4], Attachment depthAttachment);
+            Attachment colorAttachments[MRT::NUM_TARGETS], Attachment depthAttachment);
```

Comment:
Thank you :)

Reply:
Note that there's a few more hard-coded 4s in `MetalHandles.mm`, but I can fix those after you merge this

Reply:
No problem, I've now fixed up those places too.